docs/content/drafts/DropdownMenu2.mdx

```diff
@@ -0,0 +1,308 @@
+---
+component_id: dropdown_menu
+title: DropdownMenu v2
+status: Alpha
+source: https://github.com/primer/react/tree/main/src/DropdownMenu2.tsx
+storybook: '/react/storybook?path=/story/composite-components-dropdownmenu2'
+description: DropdownMenu is an ActionList-based component for creating a small menu of options to select one option from.
```

Comment:
For other components, we're using more active wording. Can we do the same here?

Maybe something like...

```suggestion
description: Use DropdownMenu to select a single option from a list of menu options.
```